discovery-provider/plugins/notifications/src/processNotifications/mappers/announcement.ts

```diff
@@ -85,10 +93,23 @@ export class Announcement extends BaseNotification<AnnouncementNotificationRow>
         )
         await this.incrementBadgeCount(userId)
       }
+
       if (
+        isLiveEmailEnabled &&
         userNotificationSettings.shouldSendEmail({ receiverUserId: userId })
       ) {
-        // TODO: Send out email
+        const notification: AppEmailNotification = {
+          receiver_user_id: userId,
+          ...this.notification
+        }
+        await sendNotificationEmail({
+          userId: userId,
+          email: userNotificationSettings.email?.[userId].email,
```

Comment:
i will leave up to you whether you want to do this or not - but would be great if we could extract this to a function in user notification settings like `userNotificationSettings.getUserEmail(userId)`. again i realize thats updating a lot of these things all over again so feel free to ignore and save for a future ticket haha

Reply:
yea good idea 👍